administrator-manual/en/release_notes.rst

```diff
@@ -35,6 +35,15 @@ Major changes on 2020-11-XX
 
 * ISO release 7.9.2009 "final" replaces any previous ISO 7.8.2003
 
+* As default, users belonging to the ``wheel`` group can now access the systems using SSH.
```

Comment:
- Explain what ``wheel`` is
- "As default" could be interpreted as "affecting existing systems" 

```suggestion
* In newly installed systems, users belonging to the ``wheel`` group (i.e. those created during the ISO installation procedure) can log in via SSH.
```